src/wallet/interface/incoming.ts

```diff
@@ -0,0 +1,139 @@
+import { State, Channel } from 'fmg-core';
+import { SignableData } from '../domain/ChannelWallet';
+import BN from 'bn.js';
+import { PlayerIndex } from '../wallet-engine/wallet-states';
+
+// FUNDING
+// =======
+
+export const FUNDING_REQUEST = 'WALLET.FUNDING.REQUEST';
+
+export const fundingRequest = (
+  channelId: string,
+  myAddress: string,
+  opponentAddress: string,
+  myBalance: BN,
```

Comment:
I remember having issues with BN right before DevCon. Do you think that we should start moving away from BN and pass around strings?

Reply:
👌